functional-tests/pom.xml

```diff
@@ -0,0 +1,164 @@
+	</properties>
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>com.jayway.jsonpath</groupId>
+				<artifactId>json-path</artifactId>
+				<version>2.2.0</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.che.selenium</groupId>
+				<artifactId>che-selenium-core</artifactId>
+				<!-- <version>${che.version}</version> -->
+				<version>6.8.0-SNAPSHOT</version>
+			</dependency>
+			<dependency>
+				<groupId>org.eclipse.che.selenium</groupId>
+				<artifactId>che-selenium-test</artifactId>
+				<!-- <version>${che.version}</version> -->
+				<version>6.8.0-SNAPSHOT</version>
+			</dependency>
+			<dependency>
+				<groupId>org.jboss.shrinkwrap.resolver</groupId>
+				<artifactId>shrinkwrap-resolver-api-maven-embedded</artifactId>
+				<version>3.1.3</version>
+			</dependency>
+			<dependency>
+				<groupId>org.jboss.shrinkwrap.resolver</groupId>
+				<artifactId>shrinkwrap-resolver-impl-maven-embedded</artifactId>
+				<version>3.1.3</version>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
+	<dependencies>
+		<dependency>
+			<groupId>com.google.guava</groupId>
+			<artifactId>guava</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>com.google.inject</groupId>
+			<artifactId>guice</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>com.google.inject.extensions</groupId>
+			<artifactId>guice-assistedinject</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>com.jayway.jsonpath</groupId>
+			<artifactId>json-path</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>com.squareup.okhttp3</groupId>
+			<artifactId>okhttp</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>javax.annotation</groupId>
+			<artifactId>javax.annotation-api</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>javax.inject</groupId>
+			<artifactId>javax.inject</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.eclipse.che.core</groupId>
+			<artifactId>che-core-api-core</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.eclipse.che.core</groupId>
+			<artifactId>che-core-api-model</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.eclipse.che.core</groupId>
+			<artifactId>che-core-api-workspace-shared</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.eclipse.che.core</groupId>
+			<artifactId>che-core-commons-lang</artifactId>
+		</dependency>
+		<!-- TEST DEPENDENCIES -->
+		<dependency>
+			<groupId>org.eclipse.che.selenium</groupId>
+			<artifactId>che-selenium-core</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.eclipse.che.selenium</groupId>
+			<artifactId>che-selenium-test</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.eclipse.jgit</groupId>
+			<artifactId>org.eclipse.jgit</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.jboss.shrinkwrap.resolver</groupId>
+			<artifactId>shrinkwrap-resolver-api-maven-embedded</artifactId>
+		</dependency>
+		<dependency>
```

Comment:
According to the build, it seems you should remove this dependency that is unused and not necessary: 

```
INFO] --- maven-dependency-plugin:3.0.1:analyze-only (analyze) @ functional-tests ---
[WARNING] Unused declared dependencies found:
[WARNING]    org.jboss.shrinkwrap.resolver:shrinkwrap-resolver-impl-maven-embedded:jar:3.1.3:compile
```